mod/Domestication/events/domestication_paragon_events.txt

```diff
@@ -136,6 +137,7 @@ leader_event = {
           save_event_target_as = new_floret
           add_trait = leader_trait_blorgfriend_floret
           add_trait = leader_trait_floret_immortal
+		  add_age = 42
```

Comment:
Putting age in last_created_leader is fine, but it should be "set_age", not "add_age"